playbooks/webapp/tasks/delete.yaml

```diff
@@ -81,6 +86,7 @@
           amazon.aws.rds_subnet_group:
             name: "{{ rds_subnet_group_name }}"
             state: absent
+          ignore_errors: true
```

Comment:
Why do we need ignore_errors  here and in the next ones? We should be able to release resources without using ignore_errors  in my opinion (when the user sets the operation to delete, we should make sure we get the exact behavior from the operation).

Reply:
Removed.